pdal/XMLSchema.cpp

```diff
@@ -431,7 +431,9 @@ bool XMLSchema::load(xmlDocPtr doc)
                     std::cerr << "Unable to fetch minimum value.\n";
                     return false;
                 }
-                dim.m_min = std::atof((const char*)n);
+                Utils::ClassicLocaleIStringStream iss((const char*)n);
+                iss >> dim.m_min;
+                iss.str("");
```

Comment:
is ``iss.str("");`` really needed ? If so, a comment in the code would be welcome. Same at line 449

Reply:
Thanks @rouault . I was afraid that the destructor of `istringstream` could conflict with `xmlFree` that is called before the constructor. Now reading carefully the documentation of `istringstream`, I see that it is created with a copy of `n`, so it is safe (if I understood correctly). I will remove that line.